src/InstructionsText.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './InstructionsText.css'
+
+function InstructionsText() {
+    return (
+        <p className="instructions-text" />
+    )
```

Comment:
You need to pass along the props here to actually render the text:

```suggestion
function InstructionsText(props) {
    return (
        <p className="instructions-text" {...props} />
    )
```